### PR TITLE
Test ambiguous concepts bug

### PR DESCRIPTION
**Pull request by dknochen**

### CODE REVIEW

File: queries/modules/io/28/apps/test.jq [hunk 1]
```diff
@@ -12,6 +12,10 @@ declare function test:is-dow30() as boolean{
   contains(credentials:credentials("MongoDB", "xbrl").db, "dow30")
 };
 
+declare function test:is-all() as boolean{
```
Comment (gislenius) [inline queries/modules/io/28/apps/test.jq]: I would rename this to make clear it's japan.

File: queries/public/test/edinet/facttable-for-component.jq [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+        ambiguous-concepts-fix: test:invoke-and-assert-deep-equal(
+          "facttable-for-component",
+          {
+            aid: "TD2014110700368",
+            role: "http://disclosure.edinet-fsa.go.jp/role/jppfs/rol_QuarterlyConsolidatedStatementOfCashFlows-indirect"
+          },
+          function($res as item*) as item* { count($res.FactTable[]) },
+          78
+        )
+    }[$is-all]
```
Comment (gislenius) [inline queries/public/test/edinet/facttable-for-component.jq]: Can you also add a test for a NIKKEI-9 filing? The bug we fixed was in all BS/IS/CS in TDNET filings.

**Comment by gislenius** [comment]
@dknochen back to you! ;-)

File: queries/public/test/edinet/facttable-for-component.jq (hunk 2 above)
Comment (dknochen) [inline queries/public/test/edinet/facttable-for-component.jq]: @gislenius I couldn't find any change due to this change in any top-9 filing. Do you have a specific example at hand? Or, was it only a guess?

**Pull request merged**